docs/DeveloperGuide.md

```diff
@@ -1,38 +1,185 @@
 # Developer Guide
+## Product Name
+**WellNUS++**
 
 ## Acknowledgements
-
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+1. Reference to AB-3 Developer Guide: https://se-education.org/addressbook-level3/DeveloperGuide.html
+2. Reference to AB-3 diagrams code: https://github.com/se-edu/addressbook-level3/tree/master/docs/diagrams
+
+## Setting up, getting started
+### Setting up the project in your computer
+Firstly, **fork** this repo, and **clone** the fork into your computer. <br>
+<br>
+If you plan to use Intellij IDEA (highly recommended): <br>
+1. **Configure the JDK**: Follow the guide 
+[[se-edu/guides] IDEA: Configuring the JDK](https://se-education.org/guides/tutorials/intellijJdk.html) 
+to ensure Intellij is configured to use **JDK 11**.
+2. **Import the project as a Gradle project**: Follow the guide 
+[[se-edu/guides] IDEA: Importing a Gradle project](https://se-education.org/guides/tutorials/intellijImportGradleProject.html) 
+to import the project into IDEA.<br>
+   **Note**: Importing a Gradle project is slightly different from importing a normal Java project.
+3. **Verify the setup:**
+   1. Run the ```wellnus.WellNus``` and try a few commands.
+   2. [Run the tests](https://se-education.org/addressbook-level3/Testing.html) to ensure they all pass.
```

Comment:
I think we can remove the link and just render `Run the tests` as normal text, since we don't have a webpage for that